Header header
float32[3] magnetic_field
float32[3] angular_velocity
float32[3] linear_acceleration
